tests/test_pmem_allocator.cpp

```diff
@@ -52,23 +52,21 @@ TEST_F(EnginePMemAllocatorTest, TestBasicAlloc) {
   // params config
   std::vector<uint64_t> num_segment_blocks{1024, 2 * 1024, 2 * 1024 * 1024};
   std::vector<uint32_t> block_sizes{32, 64};
-  std::vector<uint32_t> num_threads = {1, 16};
-
-  for (size_t i = 0; i < num_segment_blocks.size(); ++i) {
-    for (auto num_thread : num_threads) {
-      thread_manager_.reset(new ThreadManager(num_thread));
-
+  uint32_t single_thread = 1;
```

Comment:
change to thread_nr for more meaningful?

Reply:
changed to num_thread